pkg/api/frontend_logging.go

```diff
@@ -28,14 +66,121 @@ type frontendSentryEvent struct {
 	Exception *frontendSentryException `json:"exception,omitempty"`
 }
 
+func guessSourceMapLocation(sourceUrl string) (*sourceMapLocation, error) {
+	u, err := url.Parse(sourceUrl)
+	if err != nil {
+		return nil, err
+	}
+	if strings.HasPrefix(u.Path, "/public/build/") {
+		return &sourceMapLocation{
+			dir:      setting.StaticRootPath,
+			path:     path.Join("build", u.Path[len("/public/build/"):]) + ".map",
```

Comment:
Should this be a filesystem path or a network path? If it's a filesystem path, you should use `filepath.Join`.

Reply:
oops
- [ ] use `filepath.Join`